prov/tcp/src/xnet_cq.c

```diff
@@ -172,14 +173,18 @@ void xnet_report_success(struct xnet_xfer_entry *xfer_entry)
 	}
 
 	if (cq->src) {
-		ofi_cq_write_src(cq, xfer_entry->context, flags, len,
-				 xfer_entry->user_buf, data, tag,
-				 xfer_entry->src_addr);
+		ret = ofi_cq_write_src(cq, xfer_entry->context, flags, len,
+				       xfer_entry->user_buf, data, tag,
+				       xfer_entry->src_addr);
 	} else {
-		ofi_cq_write(cq, xfer_entry->context, flags, len,
-			     xfer_entry->user_buf, data, tag);
+		ret = ofi_cq_write(cq, xfer_entry->context, flags, len,
+				   xfer_entry->user_buf, data, tag);
 	}
-	if (cq->wait)
+
+	if (ret) 
+		FI_WARN(&xnet_prov, FI_LOG_CQ, "cq write failed (%s)\n",
+			fi_strerror(ret));
```

Comment:
Honestly, you could probably just cast away the return values from ofi_cq_write.  The app is the living dead at this point.  Please update the commit message to remove the reference to some coverity id and state the actual problem.  No one reading git logs is going to know what coverity 454432 means.

Are you even sure this is the correct fix for that issue?  "Value returned from a function is not checked for errors before being used"  No values being returned from ofi_cq_write were being used anyway.  Is this referring to the len value returned from xnet_msg_len()?

Reply:
It's probably a non-issue overall, but coverity flags it because it's checked every other place that that function is called. We discussed squashing the issue, but ultimately decided that the warn logging was fine.

Reply:
I'm focused on the wording "before being used" in "Value returned from a function is not checked for errors before being used".  Either the message is wrong or there's some other problem being referenced.  If the message stopped at 'errors', this change makes more sense.

Adding ```(void) ofi_cq_write(...)``` cast would indicate to coverity that the return value is intentionally being discarded.  I honestly don't care if the warning is kept instead.  My concern is whether the correct problem is being addressed.